src/grid/index.ts

```diff
@@ -8,6 +8,7 @@ import { reference } from '@dojo/framework/core/diff';
 import { Store } from '@dojo/framework/stores/Store';
 import Dimensions from '@dojo/framework/core/meta/Dimensions';
 import Resize from '@dojo/framework/core/meta/Resize';
+import './utils';
```

Comment:
Is this imported for side-effects, or was this not intentional?

Reply:
yup this needs to be imported for the module to be included in the library built.